specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-04-01-preview/examples/ApiManagementGetPrivateEndpointConnection.json

```diff
@@ -0,0 +1,29 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2021-04-01-preview",
+    "subscriptionId": "subid",
+    "privateEndpointConnectionName": "privateEndpointConnectionName"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/privateEndpointConnections/connectionName",
+        "type": "Microsoft.ApiManagement/service/privateEndpointConnections",
+        "name": "privateEndpointProxyName",
```

Comment:
>privateEndpointProxyName [](http://example.com/codeflow?start=17&length=24)

The name (and the one in ID) should match the name in the request parameter 'privateEndpointConnectionName'

Reply:
Still need to fix this one.